package.json

```diff
@@ -22,14 +23,14 @@
   "homepage": "https://github.com/salsita/foosball-rating#readme",
   "dependencies": {
     "pg": "^7.18.1",
-    "serve": "^10.1.2"
+    "serve": "^10.1.2",
+    "concurrently": "^5.1.0"
```

Comment:
this is actually a bug on master - if you install it for production (no devDependencies), then this package is missing

Reply:
oops...